src/routes/v1/ProfileRoute.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import Profile from '../../controllers/ProfileUpdateController';
+import profileChecker from '../../middlewares/CheckProfile';
+
+const router = express.Router();
+
+router.post('/signup_test', Profile.create);
+router.post('/signin_test', profileChecker, Profile.signInUser);
+router.patch('/user/:id', Profile.updateProfile);
```

Comment:
```suggestion
router.patch('/user/:id', checkToken, Profile.updateProfile);
```

Reply:
Thank you. All feedbacks have been implemented